.projenrc.ts

```diff
@@ -43,7 +43,7 @@ const project = new awscdk.AwsCdkConstructLibrary({
   authorOrganization: true,
   description: 'AWS Generative AI CDK Constructs is a library for well-architected generative AI patterns.',
   cdkVersion: CDK_VERSION,
-  projenVersion: '~0.78.8',
+  projenVersion: '~0.80.10',
   constructsVersion: '10.3.0',
   defaultReleaseBranch: 'main',
   jsiiVersion: '~5.3.0',
```

Comment:
```suggestion
  jsiiVersion: '~5.4.0',
```

Reply:
Done in [6abcd5c](https://github.com/awslabs/generative-ai-cdk-constructs/pull/564/commits/6abcd5c121c6e67f6308b7bd9bcbb1a557161876)